src/displayapp/screens/Notifications.cpp

```diff
@@ -270,6 +284,26 @@ Notifications::NotificationItem::NotificationItem(const char* title,
   lv_obj_t* alert_count = lv_label_create(container, nullptr);
   lv_label_set_text_fmt(alert_count, "%i/%i", notifNr, notifNb);
   lv_obj_align(alert_count, NULL, LV_ALIGN_IN_TOP_RIGHT, 0, 16);
+  // almost impossible to receive a real notification at time 0, so skip because it is the "no notifications" notification
+  if (timeNow != 0) {
+    auto diff = std::chrono::system_clock::from_time_t(timeNow) - std::chrono::system_clock::from_time_t(timeArrived);
+    std::chrono::minutes age = std::chrono::duration_cast<std::chrono::minutes>(diff);
+    uint32_t ageInt = static_cast<uint32_t>(age.count());
+    char timeUnit;
+    if ((ageInt / (60 * 24)) >= 1) {
+      ageInt /= (60 * 24);
+      timeUnit = 'd';
+    } else if ((ageInt / 60) >= 1) {
```

Comment:
These can be simplified.
```suggestion
    if (ageInt >= 60 * 24) {
      ageInt /= (60 * 24);
      timeUnit = 'd';
    } else if (ageInt >= 60) {
```